tests/test_authentication.py

```diff
@@ -96,36 +97,39 @@ def test_get_validated_token(self):
             self.backend.get_validated_token(str(token)).payload, token.payload
         )
 
+    @override_api_settings(
+        AUTH_TOKEN_CLASSES=("rest_framework_simplejwt.tokens.AccessToken",),
```

Comment:
Is this really needed since `AUTH_TOKEN_CLASSES` default to `("rest_framework_simplejwt.tokens.AccessToken",)`, maybe for explicitly sake.

Reply:
yes